static/app/components/profiling/flamegraph/flamegraphTooltip.tsx

```diff
@@ -46,6 +46,10 @@ export interface FlamegraphTooltipProps {
 }
 
 export function FlamegraphTooltip(props: FlamegraphTooltipProps) {
+  const start =
+    props.frame.start + Math.abs(props.flamegraphView.configSpaceTransform[6]);
+  const end = props.frame.end + Math.abs(props.flamegraphView.configSpaceTransform[6]);
```

Comment:
What is this intended to do? Can we add a comment here? Any time we directly access the matrix, it'd be nice if there was some explanation of what the intention there is.

Reply:
Let me explain this better, we basically need to ensure that any values we are reading off are properly mapped to the space. So here we should really be doing a `vec2(frame.start, frame.end).transformMat(configSpaceTransform)`